mobilecoind/strategies/drain-accounts.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2018-2021 The MobileCoin Foundation
+
+"""
+The purpose of this strategy is to drain funds from one account to another.
+This is one way to transfer tokens to a fog account, which were created after bootstrap.
+This is the case for minted tokens (which fog distro cannot transfer).
+
+It zips the source accounts and dest accounts together, and each source account sends
+its entire balance (less fee) to the dest account
+
+Example setup and usage:
+```
+    python3 drain-accounts.py --key-dir ../../../target/sample_data/master/keys/ --dest-key-dir ../../../target/sample_data/master/fog_keys/ --fee 20
+```
+"""
+import argparse
+import grpc
+import mobilecoind_api_pb2
+import mobilecoind_api_pb2_grpc
+import os
+import time
+from accounts import wait_for_accounts_sync
+from google.protobuf.empty_pb2 import Empty
+
+
+def parse_args() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--mobilecoind-host",
+                        default="localhost",
+                        type=str,
+                        help="Mobilecoind host")
+    parser.add_argument("--mobilecoind-port",
+                        default="4444",
+                        type=str,
+                        help="Mobilecoind port")
+    parser.add_argument("--key-dir",
+                        type=str,
+                        help="Path to directory of account_keys")
+    parser.add_argument("--dest-key-dir",
+                        type=str,
+                        help="Path to directory of account_keys")
+    parser.add_argument("--max-seconds",
+                        type=int,
+                        default=40,
+                        help="Number of seconds to wait for a tx to clearn")
+    parser.add_argument("--fee",
+                        type=int,
+                        default=20,
+                        help="Amount less than the balance that we attempt to send")
+    return parser.parse_args()
+
+def read_file(path):
+    with open(path, "r") as file:
+        return file.read()
+
+def run_test(stub, amount, monitor_id, dest, max_seconds):
+    resp = stub.GetBalance(
+        mobilecoind_api_pb2.GetBalanceRequest(monitor_id=monitor_id))
+    starting_balance = resp.balance
+    print("Starting balance prior to transfer:", starting_balance)
+    tx_stats = {}
+    sync_start = time.time()
+    wait_for_accounts_sync(stub, [monitor_id], 3)
```

Comment:
I think it makes more sense to wait for the sync before the initial balance check